src/components/forms/AppTextArea.vue

```diff
@@ -0,0 +1,68 @@
+<template>
+  <label v-if="label" class="block mb-1.5 font-semibold" :for="inputType"
```

Comment:
Hmm this isn't related to your task but I'm not sure how the `<label for="inputType">` and `<input id="inputType">` ever worked here, this clearly won't generate unique IDs! I'm guessing this means all our labels are broken if you try to use them to focus an input. I'll add an issue to fix this.

Reply:
Done: https://github.com/beabee-communityrm/beabee-frontend/pull/104/commits/578f343a4e4679ed35823e36a989aa613b9789b8